packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -21,28 +21,43 @@ import ErrorModal from "../common/ErrorModal";
 import { errorModalDelayMs } from "../../constants";
 import { useCreateRound } from "../../context/round/CreateRoundContext";
 import { datadogLogs } from "@datadog/browser-logs";
+import {
+  PencilIcon
+} from "@heroicons/react/solid";
+
+import {
+  CheckIcon
+} from "@heroicons/react/solid";
+import { boolean } from "yup";
 
 export const initialQuestions: QuestionOptions[] = [
   {
     title: "Email Address",
     required: true,
     encrypted: true,
     inputType: "text",
+    isInEditState: false,
```

Comment:
i think we can use component state to track `isInEditState` -- see https://github.com/gitcoinco/grants-round/compare/478--editable-applicaation-question...shavinac/478--editable

Reply:
sync'd with aditya, pushed changes to pr 👍